cpp/ps0000/p0056.so.cpp.cpp

```diff
@@ -1,10 +1,11 @@
-#include <iostream>
+#include <iostream> // 2/2

```

Comment:
- [x] used `#` instead of `@` for directive `include`
- [x] edited header file name to only `iostream`